ui/src/pages/services/index.tsx

```diff
@@ -9,20 +9,189 @@ export default function ServicesPage() {
+                quick loading times and custom functionality enhance the shopping experience for your valued customers
+                and keep them coming back for more.
+              </li>
+              <li>
+                Use what you already have, or start from square one. If you’re looking to expand your existing site into
+                e-commerce, we can integrate a custom Shopify theme right into your platform. If you’re new to the world
+                wide web, no worries--our static site development and e-commerce services go hand-in-hand.
+              </li>
+              <li>
+                Join the community of over half a million Shopify sellers, and enjoy the immense support system and
+                documentation that comes with it.
+              </li>
+            </ul>
+            <div>IMAGE</div>
+          </div>
+        </div>
+      </Section>
+      <Section>
+        <div className="grid">
+          <div>
+            <h1 className="margin-top-0">Custom Web Apps</h1>
+            <p>
+              Our custom web apps are more than just a pretty face. Equipped with powerful backend functionality along
+              with gorgeous, responsive user interfaces, these websites can accomplish anything you need with elegance
+              and speed. The Pure Func team has experience building a wide variety of custom web apps, also known as
+              full stack websites, including admin systems, analytics dashboards, social networks, and so much more. See
+              our full stack web development work in action for Level Up Tutorials (link). Get your custom QR code menu
+              started with Code My Menu.
+            </p>
+            <ul>
+              <li>
+                Push the boundaries of the menu experience. Traditional paper menus force you to balance the trade off
+                between space and content. QR code menus eliminate the space factor, allowing you to include images and
+                details for every single item.
+              </li>
+              <li>
+                Give everyone the chance to explore your delicious dishes. PDF menus are not responsive, which means it
+                takes a whole lot of zooming and scrolling to navigate them. Our QR menus are web pages, so they scale
+                perfectly on almost any mobile device.
+              </li>
+              <li>
+                Efficiently add, remove, and edit your offerings. Updating paper menus can be costly and time-consuming,
+                but websites are built to be changed. When you cook up new dishes or switch your seasonal offerings, QR
+                code menus make the transition seamless.
+              </li>
+            </ul>
+            <div>IMAGE</div>
+          </div>
+        </div>
+      </Section>
+      <Section>
+        <div className="grid">
+          <div>
+            <h1 className="margin-top-0">QR Code Menus</h1>
```

Comment:
I know the copy page said QR Code Menus but we decided recently to rebrand them as Digital Menus